gh-pages/_docs/06-05-suspicious-metrics.md

```diff
@@ -0,0 +1,14 @@
+---
+
+permalink: /docs/suspicious-metrics/
+title: "Suspicious metrics"
+
+This feature compares the values of different metrics with metric values of 241 Open Source Java projects.
+Based on this data suspicious and inconspicuous metrics are identified and corresponding suggestions can be clicked to view them.
+
+Be aware that metrics for other programming languages might not be comparable to Java Reference metric values.
+
+To activate this function it is necessary to open the global configuration menu and activate the experimental features,
```

Comment:
Since this documentation is for the stable release, I think the paragraph about how to enable experimental features can be removed.

Reply:
We can already delete this and assume it is stable already.